src/main/java/frc/robot/poseestimator/PoseEstimatorConstants.java

```diff
@@ -0,0 +1,14 @@
+package frc.robot.poseestimator;
+
+
+public class PoseEstimatorConstants {
+
+	protected static final String LOG_PATH = "PoseEstimator/";
+
+	protected static final int OBSERVATION_COUNT_FOR_POSE_CALIBRATION = 45;
```

Comment:
what?

Reply:
we take 45 vision observations and average them for a pose to use in calibration